src/main/fc/fc_core.c

```diff
@@ -278,6 +278,14 @@ static void updateArmingStatus(void)
         }
 #endif
 
+#if defined(USE_GEOZONE) && defined (USE_GPS)
+        if (geozoneIsInsideNFZ()) {
```

Comment:
We should probably expand the check to:
* do we have geozones feature enabled
* do we have any geozones defined
* do we have gps fix
* are we inside NFZ.

Reply:
geozoneIsInsideNFZ() actually does the checks, i will rename the function (geozoneIsBlockingArming()) to make it clearer. The feature check is a good idea.